packages/origin-energy-api-client/package.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "@energyweb/origin-energy-api-client",
```

Comment:
Please add `private: true` for now, we would remove it when we get it fully integrated into the rest of the system.

Reply:
fixed